muDIC/mesh/meshUtilities.py

```diff
@@ -399,11 +399,18 @@ def gen_node_positions(self):
             logger.exception("Mesh generation failed")
 
     def visualize_node(self):
-        import matplotlib.pyplot as plt
-        plt.plot(self.xnodes,self.ynodes,"o")
+        from PIL import ImageFont, Image, ImageDraw
+
+        im = Image.open(r'./example_data/ut_151.tif')
```

Comment:
This image should change, it should be updated with the first image of the current dataset.